shell/browser/plugins/plugin_utils.h

```diff
@@ -0,0 +1,34 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SHELL_BROWSER_PLUGINS_PLUGIN_UTILS_H_
+#define SHELL_BROWSER_PLUGINS_PLUGIN_UTILS_H_
+
+#include <string>
+
+#include "base/containers/flat_map.h"
+#include "base/macros.h"
+
+namespace content {
+class BrowserContext;
+}
+
+class PluginUtils {
+ public:
+  // If there's an extension that is allowed to handle |mime_type|, returns its
+  // ID. Otherwise returns an empty string.
+  static std::string GetExtensionIdForMimeType(
```

Comment:
Any necessity for the class if the only members are static ?

Reply:
just copying from //chrome/browser/plugins/plugin_utils.h